app/src/main/java/org/schabi/newpipe/MainActivity.java

```diff
@@ -66,7 +66,7 @@
 import org.schabi.newpipe.util.StateSaver;
 import org.schabi.newpipe.util.ThemeHelper;
 
-import static org.schabi.newpipe.extractor.InfoItem.InfoType.PLAYLIST;
+//Test comment test
```

Comment:
And please go through your changes and remove unused code and comments. This will speed up our review, too. :)